db-server/index.js

```diff
@@ -154,6 +154,10 @@ function createServer(db) {
   api.get('/emailRecord/:id', withIdAndBody(db.emailRecord))
   api.head('/emailRecord/:id', withIdAndBody(db.accountExists))
 
+  api.get('/totp/:id', withIdAndBody(db.getTotpToken))
```

Comment:
No biggie, but I notice most of the other token get methods don't have a "get..." prefix in the name: `sessionToken`, `keyFetchToken` etc. Should we be consistent there?